src/main/kotlin/org/jetbrains/teamcity/rest/implementation.kt

```diff
@@ -205,15 +232,15 @@ private class BuildConfigurationImpl(private val bean: BuildTypeBean,
 
     override fun fetchFinishBuildTriggers():
             List<FinishBuildTrigger> = service.buildTypeTriggers(id.stringId)
-                                              .trigger
-                                              ?.filter { it.type == "buildDependencyTrigger" }
-                                              ?.map { FinishBuildTriggerImpl(it) }.orEmpty()
+            .trigger
```

Comment:
Do you seriously think that this change makes the code more readable? If 'Reformat Code' makes the code less readable it's better to file an issue for Kotlin plugin rather than apply it to the code.